### DISCUSSION TITLE
Intelligent Octopus Charger card

### DISCUSSION THREAD

**Discussion by iPeel** [BottlecapDave/HomeAssistant-OctopusEnergy#400] [Q&A] [answered]
I moved to IO last week, and noticed today a new "Charger" card had appeared.

It seems to be related to all things IO with settings for desired state of charge, ready time etc, but what's piqued my interest is that the device info seems to reference my charge point integration ( Easee One ).

<img width="755" alt="image" src="https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/49528212/4919264c-52d7-43d1-b62f-6a2e9a5b6540">

Can I ask what the possible context of the integration is between the Octopus Integration and my EVSE?

Also, I actually have two Easee units associated with the integration, and there doesn't seem to be a way of associating the car the Octopus app is controlling and the EVSE which it is plugged into.

Is there any documentation specific to this card?

Thanks.

**Comment by iPeel** [comment, marked as answer]
OK I think I can answer my own question, it seems that the device info is the charger I said I had when I signed up to IO and not referenceing my integration, as there are subtle differences in the model name.

**Reply by BottlecapDave** [reply]
You're correct, the device info is supplied by OE. They provide both charger and target car information, but HA only supports one or the other, so I arbitrarily picked the charger. This is used mainly to group the related sensors together.
